sdk/communication/communication-chat/samples/javascript/package.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
Can we enable the `execute-samples` command in chat's package.json and make sure samples run as part of CI?